detection-rules/impersonation_github.yml

```diff
@@ -9,9 +9,9 @@ source: |
   and (
       ilike(sender.display_name, '*github*')
       or ilike(sender.email.email, '*github*')
-      or iedit_distance(sender.email.domain.sld, 'github') <= 2
+      or iedit_distance(sender.email.domain.sld, 'github') == 1
```

Comment:
`<= 1` isn't a bad idea, since it will also handle the case of different TLDs like `github.net`.
could go either way

Reply:
updated